pulp_ostree/tests/functional/api/test_import.py

```diff
@@ -354,3 +354,136 @@ def test_import_commits_same_ref(
+            "ostree.modify_ostreerepository",
+            "ostree.repair_ostreerepository",
+            "ostree.sync_ostreerepository",
+            "ostree.view_ostreerepository",
+            "ostree.add_ostreerepository",
+            "ostree.view_ostreeremote",
+            "ostree.change_ostreeremote",
+            "ostree.delete_ostreeremote",
+            "ostree.manage_roles_ostreeremote",
+            "ostree.view_ostreeremote",
+            "ostree.add_ostreeremote",
+            "ostree.view_ostreedistribution",
+            "ostree.change_ostreedistribution",
+            "ostree.delete_ostreedistribution",
+            "ostree.manage_roles_ostreedistribution",
+            "ostree.view_ostreedistribution",
+            "ostree.add_ostreedistribution",
+            "core.replicate_upstreampulp",
+            "core.view_upload",
+            "core.change_upload",
+            "core.delete_upload",
+            "core.manage_roles_upload",
+            "core.view_upload",
+            "core.add_upload",
+            "core.view_task",
+            "core.change_task",
+            "core.delete_task",
+            "core.manage_roles_task",
+            "core.view_task",
+            "core.view_taskschedule",
+            "core.manage_roles_taskschedule",
+            "core.view_taskschedule",
+            "core.download_rbaccontentguard",
+            "core.view_rbaccontentguard",
+            "core.change_rbaccontentguard",
+            "core.delete_rbaccontentguard",
+            "core.manage_roles_rbaccontentguard",
+            "core.view_rbaccontentguard",
+            "core.add_rbaccontentguard",
+            "core.view_headercontentguard",
+            "core.change_headercontentguard",
+            "core.delete_headercontentguard",
+            "core.manage_roles_headercontentguard",
+            "core.view_headercontentguard",
+            "core.add_headercontentguard",
+            "core.view_group",
+            "core.change_group",
+            "core.delete_group",
+            "core.manage_roles_group",
+            "core.view_group",
+            "core.add_group",
+            "core.view_domain",
+            "core.change_domain",
+            "core.delete_domain",
+            "core.manage_roles_domain",
+            "core.view_domain",
+            "core.add_domain",
+            "core.view_contentredirectcontentguard",
+            "core.change_contentredirectcontentguard",
+            "core.delete_contentredirectcontentguard",
+            "core.manage_roles_contentredirectcontentguard",
+            "core.view_contentredirectcontentguard",
+            "core.add_contentredirectcontentguard",
+            "core.view_compositecontentguard",
+            "core.change_compositecontentguard",
+            "core.delete_compositecontentguard",
+            "core.manage_roles_compositecontentguard",
+            "core.view_compositecontentguard",
+            "core.add_compositecontentguard",
+        ],
```

Comment:
```suggestion
        permissions=[
            "ostree.import_commits_ostreerepository",
            "ostree.add_ostreerepository",
        ],  
```

We can use just two permissions.

Reply:
Or better, we could create a user with the `ostree.ostreerepository_creator` role.

Reply:
Like this: https://github.com/pulp/pulp_container/blob/ce05037517d62b36a35e0a8c57efc7e0e46e5a45/pulp_container/tests/functional/api/test_rbac_repo_content.py#L41-L46.

Reply:
I've tested with those permissions because we've created an `ostree.admin` role for services. But for sure we can trim down the permissions.

Reply:
Also, I think we need this permission here https://github.com/pulp/pulp_ostree/pull/379/files#diff-933b628e96bbcca797e520be49a32612920c4b46068b3f45e12660ebcb68d2e9R163

Reply:
Not really. Because that permission will be automatically assigned to a respective user once she creates a repository, see the creation hooks here: https://github.com/pulp/pulp_ostree/blob/cb092dd3085530ab63d82ed03d2b9c5650ea32a8/pulp_ostree/app/viewsets.py#L184-L203.

Reply:
Please don't use permissions, but roles.

Reply:
Thanks folks. I've applied all your suggestions.